code/datums/supply_packs/food.dm

```diff
@@ -55,11 +55,140 @@
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					)
+	cost = 7
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "\improper soybeans crate"
+	group = "Food"
+
+
+/datum/supply_packs/vegetables
+	name = "carrot,corn,eggplant,chili,pumpkin,cabbage (x4)"
+	contains = list(
+					/obj/item/reagent_container/food/snacks/grown/carrot,
+					/obj/item/reagent_container/food/snacks/grown/carrot,
+					/obj/item/reagent_container/food/snacks/grown/carrot,
+					/obj/item/reagent_container/food/snacks/grown/carrot,
+					/obj/item/reagent_container/food/snacks/grown/corn,
+					/obj/item/reagent_container/food/snacks/grown/corn,
+					/obj/item/reagent_container/food/snacks/grown/corn,
+					/obj/item/reagent_container/food/snacks/grown/corn,
+					/obj/item/reagent_container/food/snacks/grown/eggplant,
+					/obj/item/reagent_container/food/snacks/grown/eggplant,
+					/obj/item/reagent_container/food/snacks/grown/eggplant,
+					/obj/item/reagent_container/food/snacks/grown/eggplant,
+					/obj/item/reagent_container/food/snacks/grown/chili,
+					/obj/item/reagent_container/food/snacks/grown/chili,
+					/obj/item/reagent_container/food/snacks/grown/chili,
+					/obj/item/reagent_container/food/snacks/grown/chili,
+					/obj/item/reagent_container/food/snacks/grown/pumpkin,
+					/obj/item/reagent_container/food/snacks/grown/pumpkin,
+					/obj/item/reagent_container/food/snacks/grown/pumpkin,
+					/obj/item/reagent_container/food/snacks/grown/pumpkin,
+					/obj/item/reagent_container/food/snacks/grown/cabbage,
+					/obj/item/reagent_container/food/snacks/grown/cabbage,
+					/obj/item/reagent_container/food/snacks/grown/cabbage,
+					/obj/item/reagent_container/food/snacks/grown/cabbage,
+					)
+	cost = 7
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "\improper Vegetable crate"
+	group = "Food"
+
+/datum/supply_packs/fruits
+	name = "banana,apple,lemon,lime,orange,berries,cherries(x4)"
```

Comment:
consider calling it a "fruits supply crate" rather than listing every single component

Reply:
done